tools/clang/unittests/SPIRV/CodeGenSpirvTest.cpp

```diff
@@ -208,6 +208,10 @@ TEST_F(FileTest, GlobalsCBufferError) {
   runFileTest("var.globals.error.hlsl", Expect::Failure);
 }
 
+TEST_F(FileTest, VarVFACEInterface) {
+  runFileTest("var.vface.interface.hlsl", Expect::Failure);
```

Comment:
I don't know it, but I suspect this one is failing because of the `Expect::Failure`

Reply:
Yes. Ah, my bad. Updated.